### ISSUE TITLE
Could you please make an up-to-date version of the package? Thanks  python-3.10.0-embed-amd64.zip

### ISSUE THREAD

**Issue by AtlantisDe** [henon/Python.Included#37]
### Could you please make an up-to-date version of the package? Thanks  python-3.10.0-embed-amd64.zip 

```c#
//current
packages embedded Python (python-3.10.0-embed-amd64.zip)

// need
python-3.10.0-embed-amd64.zip

ths very much
```

**Comment by henon** [comment]
I'll see what I can do.

**Comment by AtlantisDe** [comment]
> I'll see what I can do.

ths

**Comment by henon** [comment]
It is not gonna be easy. They changed everything in pythonnet 3.0.0. It would be much easier to update to Pythong 3.9, that can be done with the old pythonnet 2.5.1

**Comment by henon** [comment]
The changes they made will even reflect on your code. Lots of API changes in pythonnet

**Comment by henon** [comment]
Here you go: https://www.nuget.org/packages/Python.Included/3.10.0-preview1

Note, since pythonnet 3.0.0 is still preview this is also released as a preview package.

**Comment by AtlantisDe** [comment]
> Here you go: https://www.nuget.org/packages/Python.Included/3.10.0-preview1
> 
> Note, since pythonnet 3.0.0 is still preview this is also released as a preview package.

ths  i will test it
